components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -844,6 +864,14 @@ class RewardsServiceImpl : public RewardsService,
       ledger::GetContributionReportCallback callback,
       ledger::ContributionReportInfoList list);
 
+  void OnGetNotCompletedContributions(
```

Comment:
nit: Not sure this is a standard `OnGet...`... but it would read better as `OnNotCompletedContributions`

Reply:
yeah normally we just take function name and append `On`